### PR TITLE
⭐️ feat: token

### PR DESCRIPTION
**Pull request by nandiheath**
add secrets when we encounter 403 -> only available when kinto-core's KINTO_CORE_SECRET is set

### CODE REVIEW

File: src/components/pages/LoginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      >
+        {({ handleSubmit, isSubmitting }: FormikProps<any>) => {
+          return (
+            <form onSubmit={handleSubmit}>
+              <div>
+                <FormikTextField
+                  label="secret"
+                  name="secret"
+                  variant="outlined"
+                  placeholder="The KINTO_CORE_SECRET you set on kinto-core"
```
Comment (bakayolo) [inline src/components/pages/LoginPage.tsx:70]: I think we should put "you admin kinto core secret" or something like that

File: src/components/pages/LoginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    width: 140px;
+  }
+`;
+
+interface LoginProps {
+  secret: string;
+}
+
+/**
+ * This page will be revamped soon. Once we have a proper custom error page it should not be named maintenance anymore
```
Comment (bakayolo) [inline src/components/pages/LoginPage.tsx]: Got it

**Review by bakayolo — APPROVED** [review]
LGTM

File: src/components/pages/LoginPage.tsx (hunk 2 above)
Comment (nandiheath) [reply src/components/pages/LoginPage.tsx]: lol that was the code i copied from somewhere else

**Pull request merged**